tests/integration/devfile/devfile_suite_test.go

```diff
@@ -13,9 +13,8 @@ func TestDevfiles(t *testing.T) {
 
 var _ = BeforeSuite(func() {
 	const registryName string = "DefaultDevfileRegistry"
-	// Use staging OCI-based registry for tests to avoid a potential overload
 	const addRegistryURL string = "https://registry.stage.devfile.io"
 
-	helper.CmdShouldPass("odo", "registry", "delete", "DefaultDevfileRegistry", "-f")
-	helper.CmdShouldPass("odo", "registry", "add", registryName, addRegistryURL)
+	// Use staging OCI-based registry for tests to avoid a potential overload
+	helper.CmdShouldPass("odo", "registry", "update", registryName, addRegistryURL)
```

Comment:
FYI, this approach won't work I don't think:

As part of the migration from Github-based registries to registry.devfile.io, Odo has logic in it to detect if `DefaultDevfileRegistry` points to a URL other than `https://registry.devfile.io` and reverts it back accordingly if it differs

```
johncollier@jcollier-mac ~ % odo registry list
NAME                       URL                             SECURE
DefaultDevfileRegistry     https://registry.devfile.io     No
johncollier@jcollier-mac ~ % odo registry update DefaultDevfileRegistry https://registry.stage.devfile.io
? Are you sure you want to update registry "DefaultDevfileRegistry" Yes
Successfully updated registry
johncollier@jcollier-mac ~ % odo registry list
NAME                       URL                             SECURE
DefaultDevfileRegistry     https://registry.devfile.io     No
```

Reply:
@josephca Ahh, just saw the issue Jingfu opened. Nevermind my comment then, once that is fixed in Odo, this should work.

Reply:
@johnmcollier  Yup. That's why I added a comment below. :)